draft-ietf-quic-tls.md

```diff
@@ -1575,6 +1018,47 @@ version of QUIC defined in {{QUIC-TRANSPORT}} is used.
 The quic_transport_parameters extension is carried in the ClientHello and the
 EncryptedExtensions messages during the handshake.
 
+While the transport parameters are technically available prior to the
+completion of the handshake, they cannot be fully trusted until the handshake
+completes, and reliance on them should be minimized.
```

Comment:
I can't imagine how this would look like in practice. I think you'd just use them as soon as they become available, since the connection will fail if they were tampered with anyway.

Reply:
Well, some of them might be harmful in one way or another (e.g., a super-enormous flow control window).

Reply:
The main thing I can think of is a cross-protocol attack on extensions; if an attacker can make the two sides confused about what a particular extension point means (once we have them) and a client's use of one extension "looks like" something bad to the server.  But even then, it would have to be in 0-RTT data to have an effect before the handshake finishes.

Reply:
I am going to leave this as-is for now.